pkg/minikube/audit/audit.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package audit
+
+import (
+	"os"
+	"os/user"
+	"strings"
+	"time"
+
+	"github.com/spf13/viper"
+	"k8s.io/minikube/pkg/minikube/config"
+	"k8s.io/minikube/pkg/minikube/localpath"
+	"k8s.io/minikube/pkg/minikube/out/register"
+)
+
+// auditUser pulls the user flag, if empty gets the os user
+func auditUser() string {
+	u := viper.GetString(config.User)
+	if u != "" {
+		return u
+	}
+	osUser, err := user.Current()
+	if err != nil {
+		return "UNKNOWN"
+	}
+	return osUser.Username
+}
+
+// auditArgs concats the args into space delimited string
+func auditArgs() string {
```

Comment:
same naming for here. args()

+ I am curious how this would work on "minikube kubectl command" ?

do u mind sharing the audit log for this "minikube kubectl --get pods -A"

Reply:
```
{
  "data": {
    "args": "-- get pods -A",
    "command": "kubectl",
    "endTime": "2021-01-08 11:14:40.336806 -0700 MST m=+3.304018657",
    "startTime": "2021-01-08 11:14:37.069239 -0700 MST m=+0.036484991",
    "user": "spowelljr"
  },
  "datacontenttype": "application/json",
  "id": "1d695a0e-b7b3-4cb1-bf63-53741a2fd113",
  "source": "https://minikube.sigs.k8s.io/",
  "specversion": "1.0",
  "type": "io.k8s.sigs.minikube.audit"
}
```